src/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,38 @@
+package controller;
+
+import dao.DaoInterface;
+import lib.Inject;
+import model.Bet;
+import model.Human;
+
+import java.util.Scanner;
+
+public class ConsoleHandler {
+
+    @Inject
+    private static DaoInterface<Human> humanDao;
+    @Inject
```

Comment:
cs

Reply:
And it's good now
When i click Ctrl + Alt + L nothing is changing